src/components/Analysis/HistoryButtons/HistoryButtons.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from 'react';
+import { Button } from '@rmwc/button';
+
+interface HistoryButtonsProps {
+  currentMove: number;
+  lastMove: number;
+  moveForwardInHistory: (numberOfMoves: number) => void;
+  moveBackwardInHistory: (numberOfMoves: number) => void;
+  onInteraction: (newMoveNumber: number) => void;
+}
+
+export class HistoryButtons extends React.Component<HistoryButtonsProps> {
+  handleEvent = (event: KeyboardEvent) => {
+    const {
+      currentMove,
+      lastMove,
+      moveBackwardInHistory,
+      moveForwardInHistory,
+    } = this.props;
+
+    if (
+      (event.key === 'ArrowLeft' || event.key === 'Left') &&
+      this.props.currentMove !== 1
+    ) {
```

Comment:
This is formatted quite strange IMO

Reply:
Yes. Prettier has to be convinced to format it differently.

Reply:
I wasn't able to convince prettier yet.